### ISSUE TITLE
[SEToolbox-01.183.300.4] Application crash

### ISSUE THREAD

**Issue by LordMike** [midspace/SEToolbox#115]
Initially it didn't detect my SE installation (different drive). Once I provided it, files were copied and the app restarted, at which point it presented this. 01.183.300.1 worked fine :)

![image](https://user-images.githubusercontent.com/1027111/31851926-bb8288b6-b66f-11e7-9828-ad88d254c84a.png)

**Comment by LordMike** [comment]
Also. Just re-ran SEToolbox-01.183.300.1, and it detected my SE installation without issues. Perhaps this logic was changed?

**Comment by Morketh** [comment]
Space engineers is on a secondary drive. Application Ver 1.183.300.6 Same problem
![setoolbox_2017-12-05_22-05-29](https://user-images.githubusercontent.com/6291631/33645548-a70d68da-da08-11e7-92e1-7e60d1cf486e.png)

**Comment by midspace** [comment]
I've discovered in the last few days, that there is a random crash during start up occurring when SEToolbox initialises a portion of the SE codebase.
This appears to be because of a recent change made to the game.
At the moment I've got a round-about method of re-trying it until it's ready.

Until I put out a new release (which may be this Thursday due to game changes),
my only suggestion is to try opening SEToolbox again. There is a good chance it will start up fine the second time.
